### PR TITLE
refactor: Update feature flag keys

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights
This pull request includes several changes to improve feature flag management, error logging, and community event handling. It also introduces a new test case for the testimonial nudge feature.

### Feature Flag Management:
* Renamed the `SHARED_TESTIMONIALS_NUDGE_FF` feature flag to `TESTIMONIAL_AUTO_SHARE_SETTINGS_NUDGE_FEATURE_FLAG_KEY` in multiple files for consistency. [[1]](diffhunk://#diff-ad7b2ff1972a4a0904ca14cb3d12210f0fd67f990bb5dcb3f0bced90f13416c4L8-R8) [[2]](diffhunk://#diff-32009b8c4c4ae2566dccbce91953ddc78255bc038a3e82c2d5a446daea26ef90L5-R5) [[3]](diffhunk://#diff-32009b8c4c4ae2566dccbce91953ddc78255bc038a3e82c2d5a446daea26ef90L116-R116)

### Error Logging:
* Simplified error logging in the `login` method by removing the `level="error"` parameter from `log.error` calls.

### Community Event Handling:
* Improved the `delete_event` method to handle the case where the community of an event might be `None`.
* Updated the `download_data` method to remove an unused parameter from the `send_events_report` function call.
* Modified the `info` method to return `None` for the community if it does not exist.

### Testing:
* Added a comprehensive test case for the `cadmin_testimonial_nudge` feature, including setup and teardown methods, and tests for retrieving community admin names and emails, as well as preparing testimonials.

### Other Changes:
* Updated the `_get_file_url` function to return an empty string instead of `None` when an image is not present.
#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [x] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [ ] I've given my PR a meaningful title.
* [ ] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/api/store/event.py [hunk 1]
```diff
@@ -801,9 +801,11 @@ def delete_event(self, context: Context, event_id) -> Tuple[dict, MassEnergizeAP
             events = Event.objects.filter(id=event_id)
             if not events:
                 return None, InvalidResourceError()
-
-            if not is_admin_of_community(context, events.first().community.id):
-                return None, NotAuthorizedError()
+            
+            event_community = events.first().community
+            if event_community:
```
Comment (HeyMrQuaidoo) [inline src/api/store/event.py:806]: Why isn't this a single condition ? 

```
if event_community:
                if not is_admin_of_community(context, event_community.id):
```

File: src/website/views.py [hunk 2]
```diff
@@ -310,8 +310,8 @@ def _base_community_query(is_sandbox):
 
 def _get_file_url(image):
     if not image:
-        return None
-    return image.file.url if image.file else None
+        return ""
```
Comment (HeyMrQuaidoo) [inline src/website/views.py:313]: Can you add test coverage for this. @abdullai-t

**Review by HeyMrQuaidoo — CHANGES_REQUESTED** [review]

File: src/website/views.py (hunk 2 above)
Comment (abdullai-t) [reply src/website/views.py:313]: sure.

File: src/api/store/event.py (hunk 1 above)
Comment (abdullai-t) [reply src/api/store/event.py]: did that in the latest push

**Review by HeyMrQuaidoo — APPROVED** [review]

**Pull request merged**